sechub-server-core/src/main/java/com/mercedesbenz/sechub/server/core/ServerVersion.java

```diff
@@ -0,0 +1,4 @@
+package com.mercedesbenz.sechub.server.core;
+
+public record ServerVersion(String serverVersion) {
```

Comment:
I would prefer:
```suggestion
public record ServerVersion(String versionAsString) {
```

## Why?
We have a following situation:

The current implementation is equivalent to:
```java
class ServerVersion{
   public String getServerVersion() {return "0.1.0"}
}
```

it's like 
```java
class Apple {
    public String getApple(){return "i am an apple"}
}
```

means the object stands for something, but when you ask it about this it returns something else... a server version is either an object or an string...

When we use `versionAsString` we have a getter `getVersionAsString`. If we would change in future to something like having more getters (e.g. getMinor(), getMajor() etc.) we could achieve this easily but it would be still the `SeverVersion` we are asking for the information.